packages/tslint-config-loris/index.js

```diff
@@ -0,0 +1,115 @@
+module.exports = {
+    // Add additional rules from https://github.com/buzinas/tslint-eslint-rules
+    'rulesDirectory': 'node_modules/tslint-eslint-rules/dist/rules',
+    'rules': {
+        // TypeScript specific
+        'no-empty-interface': true,
+        'no-internal-module': true,
+        'typedef-whitespace': [
+            true,
+            {
+                'call-signature': 'nospace',
+                'index-signature': 'nospace',
+                'parameter': 'nospace',
+                'property-declaration': 'nospace',
+                'variable-declaration': 'nospace'
+            }
+        ],
+        'array-type': [true, 'array'],
+        'class-name': true,
+
+        // https://github.com/buzinas/tslint-eslint-rules#possible-errors
+        'trailing-comma': [true, {multiline: 'never', singleline: 'never'}],
+        'no-conditional-assignment': true,
+        'no-console': [
+            true,
+            // Need manually specify all disallowed functions.
+            'assert',
+            'dir',
+            'error',
+            'info',
+            'log',
+            'time',
+            'timeEnd',
+            'trace',
+            'warn'
+        ],
+        'no-constant-condition': [true, {checkLoops: false}],
+        'no-control-regex': true,
+        'no-debugger': true,
+        'no-duplicate-case': true,
+        'no-empty-character-class': true,
+        'no-ex-assign': true,
+        'no-extra-semi': true,
+        'no-extra-boolean-cast': true,
+        'no-inner-declarations': [true, 'both'],
+        'no-invalid-regexp': true,
+        'no-irregular-whitespace': true,
+        'no-regex-spaces': true,
+        'no-sparse-arrays': true,
+        'no-unexpected-multiline': true,
+        'no-unsafe-finally': true,
+        'use-isnan': true,
+        'valid-typeof': true,
+
+        // https://github.com/buzinas/tslint-eslint-rules#best-practices
+        'curly': true,
+        'triple-equals': true,
+        'no-arg': true,
+        // Disable this rule, because it isn't configurable
+        // https://github.com/palantir/tslint/issues/1088
```

Comment:
First issue can be remove, because since `tslint v5.5` this rule has option for empty catch blocks:

```
'no-empty': [true, 'allow-empty-catch']
```